lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/initial-journey-selection.yaml

```diff
@@ -67,3 +70,14 @@ RESET_GPG45_IDENTITY:
     next:
       targetJourney: NEW_P2_IDENTITY
       targetState: START
+
+RESET_VCS:
```

Comment:
Could we call this `RESET_FAILED_IDENTITY` so it's a bit more consistent with the other `RESET_xx` states?